yowsup/layers/protocol_groups/protocolentities/iq_result_participants_list.py

```diff
@@ -0,0 +1,44 @@
+from yowsup.structs import ProtocolEntity, ProtocolTreeNode
+from .iq_result_group import GroupResultIqProtocolEntity
+class ListParticipantsResultIqProtocolEntity(GroupResultIqProtocolEntity):
+    '''
+    <iq type="result" from="{{GROUP_ID}}" id="{{IQ_ID}}">
+        <participant jid="{{PARTICIPANT_JID}}">
+        </participant>
+    </iq>
+    '''
+
+    def __init__(self, _from, participantList):
+        super(ListParticipantsResultIqProtocolEntity, self).__init__(_from = _from)
+        self.setProps(participantList)
```

Comment:
would you like to at least test your work before submitting a pull request next time? :)